main/SwitchToggle.tsx

```diff
@@ -1,10 +1,8 @@
 import {Animated, StyleProp, TouchableOpacity, ViewStyle} from 'react-native';
-import {DoobooTheme, light} from './theme';
 import React, {ReactElement, useEffect, useState} from 'react';
 
-import {isEmptyObject} from './utils';
 import styled from '@emotion/native';
-import {withTheme} from '@emotion/react';
+import {useTheme} from './theme/ThemeProvider';
 
 interface Styles {
   containerStyle?: ViewStyle;
```

Comment:
You may remove `prefix` `Style` when it is grouped under `Styles`.

Reply:
Ok. I'll remove it in other PR. (because it is not related to this PR.)